【PaddlePaddle Hackathon】40. 在 Paddle 中新增 paddle.nn.UpsamplingBilinear2D 的测试用例

asdfefbhjy opened the pull request:
PR types
Others

PR changes
add  framework/api/nn/test_upsampling_bilinear_2d.py

Describe
Task: https://github.com/PaddlePaddle/Paddle/issues/35965
用numpy重新实现paddle.nn.UpsamplingBilinear2D的功能，增加了测试case

asdfefbhjy committed:
add-pytest-of-paddle.nn.UpsamplingBilinear2D 0.1

kolinwei commented on framework/api/nn/test_upsampling_bilinear_2d.py:
```diff
@@ -0,0 +1,273 @@
+    data_format = "MCHW"
+    obj.exception(
+        etype=ValueError, mode="python", data=x, size=size, scale_factor=scale_factor, data_format=data_format
+    )
```
补充size与scale_factor都设置情况下的case。参考文档"size 和 scale_factor 至少要设置一个。 size 的优先级高于 scale_factor "

asdfefbhjy replied:
当size和scale_factor都设置时，会报错：
ValueError: Only one of size or scale_factor should be defined.
所以我只在最后又增加了一个exception case

asdfefbhjy committed:
add one more exception case

asdfefbhjy committed:
Update test_upsampling_bilinear_2d.py

add one more exception case: " ValueError: Only one of size or scale_factor should be defined."

asdfefbhjy committed:
Merge branch 'develop' into task-40-add-pytest-of-paddle.nn.UpsamplingBilinear2D

asdfefbhjy committed:
Delete test_upsampling_bilinear_2d.py

asdfefbhjy committed:
add one more exception case

DDDivano committed:
Merge branch 'develop' into task-40-add-pytest-of-paddle.nn.UpsamplingBilinear2D

DDDivano committed:
Merge branch 'develop' into task-40-add-pytest-of-paddle.nn.UpsamplingBilinear2D

DDDivano committed:
Merge branch 'develop' into task-40-add-pytest-of-paddle.nn.UpsamplingBilinear2D

asdfefbhjy committed:
Merge branch 'develop' into task-40-add-pytest-of-paddle.nn.UpsamplingBilinear2D
